content/documentation/multiformat-output.md

```diff
@@ -1,12 +1,222 @@
+
+If you make further edits to your project and would like to preview them you will need to repeat this process.
+
+1. Delete the old compressed `site` folder.
+
+2. Run `quire site` again (your files will be automatically overwritten.)
+
+3. Compress the newly updated `site` folder.
+
+4. In Netlify, navigate to “Deploys” at the top of the page. You will see a blank space that reads, “Need to update your site." Simply drag-and-drop your new compressed `site` folder here and your link will be automatically updated.
+
+### Continuous Deploy with Netlify
+
+*This deployment option requires a basic understanding of how to create a repository, and commit and merge changes in GitHub. If you are new to GitHub, we recommend starting with [GitHub Docs](https://docs.github.com/en/github) to learn by topic or checking out the [GitHub for Quire](/documentation/github) section of our documentation. We also encourage you to check out Coding Train’s video series is a great resource as well [Git and Github for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV).*
+
+By keeping your project files on GitHub and linking them directly to your Netlify account, any time you merge changes in Github, your preview link will be automatically updated. This process requires a few extra steps to get set-up, but will save time in the long run. This process also works with GitLab and Bitbucket.
+
+1. If you haven't already, create a repository for your project on GitHub.
+
+2. To proceed with deployment, you will need to add two files to your repository: `netlify.toml` and `package.json`. [Download them](/downloads/site-deploy.zip), add them to your repo, and merge changes before proceeding.
+
+3. Log in to [Netlify](https://app.netlify.com/). On the “Team Overview” page, click the button that says “New Site from Git.”
+
+4. Next you'll want to “Connect to Git Provider.”
+
+5. If you see “no repositories found” you will be prompted to configure Netlify on GitHub.
+
+6. Once this configuration is complete, choose the repository you would like to preview.
+
+7. Set the "build command" as `npm run build` and the "publish directory" as `site`.  
+
+8. Hit "Deploy Site." Depending on the size of your project, this may take a few moments. Follow along with the build process (and check for errors) by navigating to "Publication Deploys" section and scrolling down to the "Deploy Log".
+
+9. You will be given a default URL to preview your project. You can rename this URL by navigating to “Site settings” and changing the site name. (You also have the option to buy a domain or set-up a domain you already own.)
+
+10. Should you need to make any updates to your site, simply merge the changes and Netlify will automatically update your preview link. You can check "Production Deploys" in the site overview section to these track changes.
```

Comment:
I don't think it's a particular issue here, but a reminder that we should generally avoid words like "simply" when it might not feel simple to our user.